ui/pages/onboarding-flow/welcome/index.scss

```diff
@@ -1,5 +1,3 @@
-@import "./react-gallery/carousel.min.css";
-
```

Comment:
Need to figure out how to handle this external stylesheet as it's effecting the build